find/scripts/extract_download_logs.py

```diff
@@ -72,33 +70,42 @@ def extract_ids_from_message(message: List[List[dict]]) -> List[str]:
     return account_ids
 
 
-def fetch_user_data(user_ids: List[str]) -> dict:
+def get_email_addresses_for_user_ids(user_ids: List[str]) -> dict:
     """
-    Fetch user data from account-store API, using users' azure_ad_subject_id-s
+    Fetch user data from account-store API, using users' account_id-s
     args: user_ids: list of user_ids
     returns: dict of user_id: email_address
     """
+
+    account_store_api_host = os.getenv("ACCOUNT_STORE_API_HOST", "http://localhost:3003")
     responses = {}
```

Comment:
Can we call this `email_addresses` or something more descriptive?

Reply:
Renamed the variable, as requested.